www/admin/site-edit.php

```diff
@@ -211,10 +211,11 @@
 $themelist = [];
 $themes    = scandir(nZEDb_WWW . "/themes");
 foreach ($themes as $theme) {
-	if (strpos($theme, ".") === false && is_dir(nZEDb_WWW . "/themes/" . $theme)) {
+	if (strpos($theme, ".") === false && is_dir(nZEDb_THEMES . $theme) && ucfirst($theme) === $theme) {
```

Comment:
I don't see a need to revert it, unless you can suggest a better alternative.
It *will* be documented in ChangeLog and ReleaseNotes. It is true nothing would be displayed to say why themes might stop, but checking the ChangeLog/ReleaseNotes should quickly solve that if they hadn't read them before hand, as any site admin should.
I chose that method as it made semantic sense to me - themes are named, and as proper nouns they should have a capital first letter. The additional intent is that we can use lower-cased names as internal themes. So 'admin' can be moved to be a theme and not part of shared, which seems weird to me.
In the long run, I think this will make themes more consistent and easier for all.